packages/swap/src/routes/v2/quote.ts

```diff
@@ -256,16 +195,23 @@ export const generateQuotes = async ({
     throw quoteResult.reason;
   }
 
-  const dcaQuote = dcaQuoteResult.value;
   const quote = getFulfilledResult(quoteResult, null);
   const boostedQuote = getFulfilledResult(boostedQuoteResult, null);
+  const dcaQuote = getFulfilledResult(dcaQuoteResult, null) as DCAQuote | null;
```

Comment:
this can be a DCA quote or a regular quote depending on the number of chunks

Reply:
if we check the number of chunks before hand this can only be a `DCAQuote` ?